### ISSUE TITLE
ConcurrentModificationException while executing doInBackground()

### ISSUE THREAD

**Issue by litrik** [Pushwoosh/pushwoosh-android-sdk#151]
Versions
 - com.pushwoosh:pushwoosh:6.1.0
 - com.pushwoosh:pushwoosh-firebase:6.1.0

Device: HUAWEI nova 5T
Android 10

I have seen the following crash arrive in Crashlytics:

```
Fatal Exception: java.lang.RuntimeException: An error occurred while executing doInBackground()
       at android.os.AsyncTask$AsyncFutureTask.done(AsyncTask.java:429)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
       at java.util.concurrent.FutureTask.run(FutureTask.java:271)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:292)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:929)
Caused by java.util.ConcurrentModificationException
       at java.util.LinkedHashMap$LinkedHashIterator.nextNode(LinkedHashMap.java:775)
       at java.util.LinkedHashMap$LinkedEntryIterator.next(LinkedHashMap.java:808)
       at java.util.LinkedHashMap$LinkedEntryIterator.next(LinkedHashMap.java:806)
       at org.json.JSONObject.writeTo(JSONObject.java:732)
       at org.json.JSONObject.toString(JSONObject.java:701)
       at com.pushwoosh.internal.utils.JsonUtils.jsonToMap(:10)
       at com.pushwoosh.internal.utils.JsonUtils.jsonToMap(:1)
       at com.pushwoosh.inapp.view.a.a(:12)
       at com.pushwoosh.inapp.view.a.doInBackground(:2)
       at android.os.AsyncTask$3.call(AsyncTask.java:389)
       at java.util.concurrent.FutureTask.run(FutureTask.java:266)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:292)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:929)
```

**Comment by wfhm** [comment]
Checking it.

**Comment by wfhm** [comment]
Fixed with 6.1.1.
